doc/release/release_template.rst

```diff
@@ -37,4 +37,4 @@ Deprecations
 
 
 Contributors to this release
-----------------------------
+----------------------------
```

Comment:
Due to a misunderstanding of one of your perivious comments, I modified this file, and then I was asked to remove it. I did not realize this would be considered to be a change.